src/xcode/ENA/ENA/Source/Models/Exposure/ExposureManager.swift

```diff
@@ -186,7 +186,13 @@ final class ENAExposureManager: NSObject, ExposureManager {
 							Log.error("AppDelegate is not of Type AppDelegate, this should not happen!", log: .background)
 							return
 						}
-						Log.info("Starting backgroundTask via Bluetooth", log: .background)
+						
+						guard !CWAHibernationProvider.shared.isHibernationState else {
```

Comment:
Do we really need this code? 

We are already calling appDelegate.taskExecutionDelegate.executeENABackgroundTask below and you have the same code there.

Reply:
The initial reason was that `init` and `executeENABackgroundTask` are both `internal` accessible. But you're right, after `executeENABackgroundTask` isn't any further code. So I remove the `guard` in the `init` block.